### DISCUSSION TITLE
[Help] Textures

### DISCUSSION THREAD

**Discussion by EndiKM** [bridge-core/bridge.#44] [Q&A] [answered]
Hello, I have a question if one can put two textures on one block, for example wood texture because I created a block of wood on which I would like to put two textures but I do not know how and if you can.

I did the texture on the sides but I don't know how to give the texture to on the top and on bottom.

**Comment by solvedDev** [comment, marked as answer]
Just take a look at the vanilla logs to see how to do it (blocks.json file) :)
